src/renderer/Presentational/ModalManager/ResetConfigModal.tsx

```diff
@@ -0,0 +1,68 @@
+import { useState } from 'react';
+import { setModalState } from '../../state/modal';
+import electron from '../../electronGlobal';
+import { useAppDispatch } from '../../state/hooks';
+import { Modal } from '../../Generics/redesign/Modal/Modal';
+import { modalOnChangeConfig, ModalConfig } from './modalUtils';
+import ResetConfigWrapper from '../ResetConfigModal/ResetConfigWrapper';
+
+type Props = {
+  modalOnClose: () => void;
+};
+
+export const ResetConfigModal = ({ modalOnClose }: Props) => {
+  const [modalConfig, setModalConfig] = useState<ModalConfig>({});
+  const dispatch = useAppDispatch();
+  const modalTitle = 'Are you sure you want to reset config to defaults?';
```

Comment:
Minor: I think we should keep "settings" or "config" consistent in the UI. So should we change "config" -> "settings" for any user facing text?

+ could you add strings in the translation files here?

Reply:
+ adding "all" to convey significance to user "...reset all node settings..."